navigation/perfect_memory_store_hog.h

```diff
@@ -28,21 +28,24 @@ template<typename Differencer = AbsDiff>
 class HOG
 {
 public:
-    HOG(const cv::Size unwrapRes,
-                     const int HOGPixelsPerCell = 10,
-                     int HOGOrientations = 8)
-      : m_HOGDescriptorSize(unwrapRes.width * unwrapRes.height *
-                            HOGOrientations / (HOGPixelsPerCell * HOGPixelsPerCell))
-      , m_Differencer(m_HOGDescriptorSize)
+    HOG(const cv::Size unwrapRes, const cv::Size &blockSize, int numOrientations)
```

Comment:
*Tiny* nit, but shouldn't ``unwrapRes`` also be passed by reference if the other ``cv::Size`` params are (also in other ctor)?